app/views/components/config/DissociateSummaryConfig.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.components.config
+
+import config.AppConfig
+import javax.inject.Inject
+import play.api.mvc.Call
+
+class DissociateSummaryConfig @Inject()(appConfig: AppConfig) {
```

Comment:
How about instead of creating several ...Config classes, create something like a Navigator in Declarations service?

Reply:
I thought the idea was to limit the areas of concern - i.e. one class to determine the behaviour of one view.   The code will be easy to remove.  Delete the feature flag in AppConfig and fix all the compilation errors.

Reply:
Didn't think about it in this way. I agree that with this in mind it looks more sensible.